pkg/resourcemonitor/excludelist.go

```diff
@@ -0,0 +1,36 @@
+package resourcemonitor
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// ExcludeResourceList contains a list of resources to ignore during resources scan
+type ExcludeResourceList struct {
+	excludeList map[string]sets.String
+	nodeName    string
```

Comment:
Umm, I don't think that argument holds as the `nodeName` is not actually used for anything in `resourcemonitor`, afaics 🤔

Reply:
> Umm, I don't think that argument holds as the nodeName is not actually used for anything in resourcemonitor, afaics thinking

We are using it in `IsExcluded()` 
https://github.com/k8stopologyawareschedwg/node-feature-discovery/blob/exclude_list/pkg/resourcemonitor/excludelist.go#L27

> I think it would be cleaner to only store configuration of this particular node in the config struct.
I see your point. I can fix that

Reply:
> We are using it in IsExcluded()

Umm, yes, but if you only store the exlude list relevant for this node you never need to check the nodeName 🧐 What I'm trying to say that It's totally useless information in the config of this node and useless extra work and loc to check it.